### PR TITLE
design-philosophy.rst

### PR DESCRIPTION
**Pull request by kalinovka**
Could you publish a chapter about Suru desighn principles to Humanguide section ?

This chapter was published in original Ubuntu Touch Documentation, that is now available only at WebArchive.
I converted original text into rst format.

### CODE REVIEW

File: humanguide/design-concepts/design-philosophy.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The Ubuntu interface has been designed according to a philosophy called Suru.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubuntu we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translates into our design philosophy called Suru.
```
Comment (doniks) [inline humanguide/design-concepts/design-philosophy.rst:13]: We're not Canonical. It's not prudent to say "Here at Ubuntu"

Comment (doniks) [reply humanguide/design-concepts/design-philosophy.rst:13]: has been translates ... grammar

File: humanguide/design-concepts/design-philosophy.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- Suru stems from the Ubuntu brand values alluding to Japanese culture.
+
+- The design of Suru is inspired by origami, because it give us a solid and tangible foundation.
```
Comment (doniks) [inline humanguide/design-concepts/design-philosophy.rst:17]: There's a lot of us/our which I think would have to be reformulated to show the historic origin from some other party (Canonical) that is not us/our

**Review by doniks — CHANGES_REQUESTED** [review]

**Comment by doniks** [comment]
It doesn't build: https://ci.ubports.com/blue/organizations/jenkins/docs.ubports.com/detail/PR-457/1/pipeline

**Comment by doniks** [comment]
Above and beyond the more technical remarks above ... I simply do not know whether this still applies ... @cibersheep could you way in here?

File: humanguide/design-concepts/design-philosophy.rst (hunk 2 above)
Comment (kalinovka) [reply humanguide/design-concepts/design-philosophy.rst:17]: Hi! Thanks for your reply. Perhaps, the text sholud be corrected or rewritten. I agree that all Canonical stuff should be reformulated. I don't like that there is a link in Docementation to this chapter, and it leads to page existing only at Webarchive. So I converted it into rst text. Certainly, I can rewrite the whole chapter.

File: humanguide/design-concepts/design-philosophy.rst (hunk 1 above)
Comment (kalinovka) [reply humanguide/design-concepts/design-philosophy.rst:13]: > 
> 
> has been translates ... grammar

yep..original text contains errors.

File: humanguide/design-concepts/design-philosophy.rst (hunk 2 above)
Comment (doniks) [reply humanguide/design-concepts/design-philosophy.rst:17]: Oh I wasn't aware we have such a link. Shouldn't that be changed with this PR as well then?

Comment (doniks) [reply humanguide/design-concepts/design-philosophy.rst:17]: But, I would really like to hear from someone like @cibersheep first, whether this is indeed still our philosophy. Before you spend too much time editing... Hopefully there *is* someone who can answer that :D

**Comment by cibersheep** [comment]
> Above and beyond the more technical remarks above ... I simply do not know whether this still applies ... @cibersheep could you way in here?

I would say that the philosophy of the design is the same, as we keep the Suru vision. So, if you have the time to adapt the test to the docs, go for it ;)

Thanks

**Comment by doniks** [comment]
Hey @kalinovka are you still looking into this?

**Comment by kalinovka** [comment]
> Hey @kalinovka are you still looking into this?

Hi! I am still here. I am ready to make all necessary corrections.

File: humanguide/index.rst [hunk 3]
```diff
@@ -1,7 +1,58 @@
...
+Ubuntu Touch follows a modern and convergent design philosophy called Suru.
+We aim to keep developing the design started by Canonical.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubports we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translated into our design philosophy called Suru.
```
Comment (doniks) [inline humanguide/index.rst:13]: I'd suggest "into our design" -> "into the design"

File: humanguide/index.rst [hunk 4]
```diff
@@ -1,7 +1,58 @@
...
+
+- Suru items from the UTouch values alluding to Japanese culture.
```
Comment (doniks) [inline humanguide/index.rst:15]: I think that must be a historical Canonical/Ubuntu reference and I think there it is best to keep the original formulation, so "from the UTouch values" -> "from the Ubuntu values"

File: humanguide/index.rst [hunk 5]
```diff
@@ -1,7 +1,58 @@
...
+
+- The design of Suru is inspired by origami, because it give us a solid and tangible foundation.
```
Comment (doniks) [inline humanguide/index.rst:17]: give -> gives

File: humanguide/index.rst [hunk 6]
```diff
@@ -1,7 +1,58 @@
...
+
+- Paper can be used in all areas of the brand in two and three dimensional forms, because it is transferable and diverse.
```
Comment (doniks) [inline humanguide/index.rst:19]: let's say "brand" -> "design"

File: humanguide/index.rst [hunk 7]
```diff
@@ -1,7 +1,58 @@
...
+
+Translated in design
+--------------------
+
+Suru brings a precise yet organic structure to the UTouch interface. The sharp lines and varying levels of transparency evoke the edges and texture of paper. All elements are placed deliberately, with the express aim of being easy for the user to identify and use.
```
Comment (doniks) [inline humanguide/index.rst:24]: we never call it "UTouch" anywhere. the official name is "Ubuntu Touch"

File: humanguide/index.rst [hunk 8]
```diff
@@ -1,7 +1,58 @@
...
+
+When using a small layout, the information and functionality is folded into a compact object that that can be refolded to expose different areas. As the layout size increases, the object can become progressively larger, allowing more of the information and functionality to be exposed at any one time.
+
+Origami
+-------
+
+.. figure:: /_static/images/humanguide/366w_DesignPhilosophy_origami.png
+
+Origami has long been associated with good fortune and represents the visual style for the Ubuntu Phone. Origami folds are used to define the design.
```
Comment (doniks) [inline humanguide/index.rst:33]: "Ubuntu Phone" is somehow never used anywhere anymore, let's make this:

"the visual style for Ubuntu Touch."

**Review by doniks — CHANGES_REQUESTED** [review]
thanks for the update. just a few more small edits below please

**Review by doniks — APPROVED** [review]

**Pull request merged**